sample/alpha/build.gradle.kts

```diff
@@ -7,4 +7,6 @@ dependencies {
     implementation(project(":sample:beta"))
     implementation(project(":sample:container:gama"))
     implementation(project(":sample:container:delta"))
+
+    testImplementation(project(":sample:test"))
```

Comment:
Added for testing ignoring option.
I executed command below and checked README.md hasn't changed.

```
./gradlew :sample:createModuleGraph
```